### PR TITLE
Fix user type text on creation

### PR DESCRIPTION
**Pull request by tristanhcole**
**Describe the Pull Request**
We changed how we handle user type creation, from a URL param to redux form state. Fixes the text/button so that "Add a [ACCOUNT_TYPE]" is consistent.

Screen Shot 2020-07-31 at 4 29 12 pm

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver

### CODE REVIEW

**Comment by tristanhcole** [comment]
[Adding Vendor prompts a message mentioning "Beneficiary"](https://trello.com/c/eVI8IJFQ/475-adding-vendor-prompts-a-message-mentioning-beneficiary)

File: app/client/components/user/CreateUser.tsx [hunk 1]
```diff
@@ -30,16 +32,19 @@ interface OuterProps {
   isVendor: boolean;
 }
 
-declare global {
-  interface Window {
-    BENEFICIARY_TERM: string;
```
Comment (tristanhcole) [inline app/client/components/user/CreateUser.tsx:35]: we should really audit how we are handling this across the software - it's not consistent

Comment (enjeyw) [reply app/client/components/user/CreateUser.tsx:35]: Agreed that this needs some thought - assuming that setting the term for a beneficiary is distinct from doing translations, what other handling methods do we have? and where?

Comment (tristanhcole) [reply app/client/components/user/CreateUser.tsx:35]: Let's resolve this with https://trello.com/c/bGS17Tyz/511-consistent-and-explicit-labelling-of-user-types-across-entire-platform

**Comment by tristanhcole** [comment]
> So the reason this was a url param rather than redux in the first place is that it allowed u to send a direct link to someone with the correct account type set - good for when u've got low-literacy team members and u just want them to fill in a couple of fields.

So is this deal breaker? I could update so the URL query string updates and handles correctly

**Comment by enjeyw** [comment]
> > So the reason this was a url param rather than redux in the first place is that it allowed u to send a direct link to someone with the correct account type set - good for when u've got low-literacy team members and u just want them to fill in a couple of fields.
> 
> So is this deal breaker? I could update so the URL query string updates and handles correctly

Nah I don't think it is a deal breaker, though am subsequently questioning (to ur point i guess) the extent to which this PR aligns with ( https://trello.com/c/bGS17Tyz/511-consistent-and-explicit-labelling-of-user-types-across-entire-platform)

**Comment by tristanhcole** [comment]
> Nah I don't think it is a deal breaker, though am subsequently questioning (to ur point i guess) the extent to which this PR aligns with ( https://trello.com/c/bGS17Tyz/511-consistent-and-explicit-labelling-of-user-types-across-entire-platform)

I think that is a separate PR/audit

**Comment by enjeyw** [comment]
> > Nah I don't think it is a deal breaker, though am subsequently questioning (to ur point i guess) the extent to which this PR aligns with ( https://trello.com/c/bGS17Tyz/511-consistent-and-explicit-labelling-of-user-types-across-entire-platform)
> 
> I think that is a separate PR/audit

Ok makes sense, let's merge this in then

**Pull request merged**